packages/flutter_tools/test/general.shard/plugin_parsing_test.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter_tools/src/base/common.dart';
```

Comment:
The presubmit analyzer pass is saying that this import is unused and can be removed.

Reply:
That's right, it was used before fixing the nit. Removed.